packages/network/src/logic/Node.ts

```diff
@@ -172,6 +174,7 @@ export class Node extends EventEmitter {
     start(): void {
         this.logger.trace('started')
         this.trackerConnector.start()
+        this.connectionCleanUpInterval = this.startConnectionCleanUpInterval(2 * 60 * 1000)
```

Comment:
Recommend adding `clearInterval(this.connectionCleanUpInterval!)` before setting it for good measure. Accidentally having multiple intervals running can silently cause all sorts of weird problems and it's easy to prevent entirely if you *always* clear the timeout before you set a timeout/interval